src/item.cpp

```diff
@@ -3531,7 +3531,7 @@ void item::armor_info( std::vector<iteminfo> &info, const iteminfo_query *parts,
                 temp = sbp->opposite;
             } else {
                 // if it doesn't have an opposite print it alone and continue
-                coverage += _( " The <info>" + sbp->name + "</info>" );
+                coverage += string_format( _( "The <info> %s </info>" ), sbp->name );
```

Comment:
```suggestion
                coverage += string_format( _( " The <info>%s</info>" ), sbp->name );
```

I think the leading space was there to separate the addition from whatever may already be in the `coverage` variable. There were also no spaces inside the tag before.